tgui/packages/tgui/interfaces/DetectiveBoard/Evidence.tsx

```diff
@@ -38,6 +38,8 @@ export function Evidence(props: EvidenceProps) {
     null,
   );
 
+  const [randomRotation] = useState(Math.random() * 2 - 1);
```

Comment:
you won't need a state for this unless you're intent on changing it via user interaction or something